source/agora/node/Fee.d

```diff
@@ -0,0 +1,157 @@
+    assert(calculateDataFee(60, 100) == Amount(8_200_000L));
+    assert(calculateDataFee(70, 100) == Amount(10_100_000L));
+    assert(calculateDataFee(80, 100) == Amount(12_300_000L));
+    assert(calculateDataFee(90, 100) == Amount(14_600_000L));
+    assert(calculateDataFee(100, 100) == Amount(17_200_000L));
+    assert(calculateDataFee(110, 100) == Amount(20_000_000L));
+    assert(calculateDataFee(120, 100) == Amount(23_200_000L));
+    assert(calculateDataFee(130, 100) == Amount(26_700_000L));
+    assert(calculateDataFee(140, 100) == Amount(30_600_000L));
+    assert(calculateDataFee(150, 100) == Amount(34_800_000L));
+    assert(calculateDataFee(160, 100) == Amount(39_500_000L));
+    assert(calculateDataFee(170, 100) == Amount(44_700_000L));
+    assert(calculateDataFee(180, 100) == Amount(50_500_000L));
+    assert(calculateDataFee(190, 100) == Amount(56_900_000L));
+    assert(calculateDataFee(200, 100) == Amount(63_900_000L));
+    assert(calculateDataFee(210, 100) == Amount(71_700_000L));
+    assert(calculateDataFee(220, 100) == Amount(80_300_000L));
+    assert(calculateDataFee(230, 100) == Amount(89_700_000L));
+    assert(calculateDataFee(240, 100) == Amount(100_200_000L));
+    assert(calculateDataFee(250, 100) == Amount(111_800_000L));
+    assert(calculateDataFee(260, 100) == Amount(124_600_000L));
+    assert(calculateDataFee(270, 100) == Amount(138_800_000L));
+    assert(calculateDataFee(280, 100) == Amount(154_400_000L));
+    assert(calculateDataFee(290, 100) == Amount(171_700_000L));
+    assert(calculateDataFee(300, 100) == Amount(190_900_000L));
+    assert(calculateDataFee(310, 100) == Amount(212_000_000L));
+    assert(calculateDataFee(320, 100) == Amount(235_300_000L));
+    assert(calculateDataFee(330, 100) == Amount(261_100_000L));
+    assert(calculateDataFee(340, 100) == Amount(289_600_000L));
+    assert(calculateDataFee(350, 100) == Amount(321_200_000L));
+    assert(calculateDataFee(360, 100) == Amount(356_000_000L));
+    assert(calculateDataFee(370, 100) == Amount(394_500_000L));
+    assert(calculateDataFee(380, 100) == Amount(437_000_000L));
+    assert(calculateDataFee(390, 100) == Amount(484_000_000L));
+    assert(calculateDataFee(400, 100) == Amount(536_000_000L));
+    assert(calculateDataFee(410, 100) == Amount(593_400_000L));
+    assert(calculateDataFee(420, 100) == Amount(656_900_000L));
+    assert(calculateDataFee(430, 100) == Amount(727_000_000L));
+    assert(calculateDataFee(440, 100) == Amount(804_500_000L));
+    assert(calculateDataFee(450, 100) == Amount(890_200_000L));
+    assert(calculateDataFee(460, 100) == Amount(984_800_000L));
+    assert(calculateDataFee(470, 100) == Amount(108_9500_000L));
+    assert(calculateDataFee(480, 100) == Amount(1_205_100_000L));
+    assert(calculateDataFee(490, 100) == Amount(1_332_900_000L));
+    assert(calculateDataFee(500, 100) == Amount(1_474_100_000L));
+
+    // When the factor is 200
```

Comment:
Why do we test every 10 values ? That sounds very superfluous, and as soon as this will be updated, it'll be even less fun than the quorum tests.